### ISSUE TITLE
Uninitialised values in sixup

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#2421]
Found with memcheck:
```
On server start:
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1E03CE: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1781)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1E0628: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1815)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1EB7BC: int maximum<int>(int, int) (math.h:79)
==463034==    by 0x1E06C3: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1822)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1EB7BC: int maximum<int>(int, int) (math.h:79)
==463034==    by 0x1E076A: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1824)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x5061B7B: __vfprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x5074A79: __vsnprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x1C315D: str_format (system.c:2351)
==463034==    by 0x1E04C9: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1795)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Use of uninitialised value of size 8
==463034==    at 0x5047B3B: _itoa_word (in /usr/lib/libc-2.31.so)
==463034==    by 0x50614EF: __vfprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x5074A79: __vsnprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x1C315D: str_format (system.c:2351)
==463034==    by 0x1E04C9: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1795)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x5047B4C: _itoa_word (in /usr/lib/libc-2.31.so)
==463034==    by 0x50614EF: __vfprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x5074A79: __vsnprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x1C315D: str_format (system.c:2351)
==463034==    by 0x1E04C9: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1795)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x5062428: __vfprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x5074A79: __vsnprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x1C315D: str_format (system.c:2351)
==463034==    by 0x1E04C9: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1795)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x5061666: __vfprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x5074A79: __vsnprintf_internal (in /usr/lib/libc-2.31.so)
==463034==    by 0x1C315D: str_format (system.c:2351)
==463034==    by 0x1E04C9: CServer::CacheServerInfo(CServer::CCache*, int, bool) (server.cpp:1795)
==463034==    by 0x1E188A: CServer::UpdateServerInfo(bool) (server.cpp:2079)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1EB7BC: int maximum<int>(int, int) (math.h:79)
==463034==    by 0x1E1043: CServer::CacheServerInfoSixup(CServer::CCache*, bool) (server.cpp:1977)
==463034==    by 0x1E18FC: CServer::UpdateServerInfo(bool) (server.cpp:2082)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x1EB7BC: int maximum<int>(int, int) (math.h:79)
==463034==    by 0x1E108C: CServer::CacheServerInfoSixup(CServer::CCache*, bool) (server.cpp:1979)
==463034==    by 0x1E18FC: CServer::UpdateServerInfo(bool) (server.cpp:2082)
==463034==    by 0x1E6B14: CServer::ConchainSpecialInfoupdate(IConsole::IResult*, void*, void (*)(IConsole::IResult*, void*), void*) (server.cpp:3174)
==463034==    by 0x18FB53: CConsole::Con_Chain(IConsole::IResult*, void*) (console.cpp:1173)
==463034==    by 0x17F882: CConsole::ExecuteLineStroked(int, char const*, int, bool) (console.cpp:506)
==463034==    by 0x17FB89: CConsole::ExecuteLine(char const*, int, bool) (console.cpp:560)
==463034==    by 0x17FE92: CConsole::ExecuteFile(char const*, int, bool, int) (console.cpp:606)
==463034==    by 0x1E86C9: main (server.cpp:3507)
==463034== 

On join:
[2020-06-30 16:58:01][server]: player has entered the game. ClientID=0 addr=<{192.168.0.2:63032}> sixup=1
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x2198CC: CGameContext::SendChat(int, int, char const*, int, int) (gamecontext.cpp:408)
==463034==    by 0x21D498: CGameContext::OnClientEnter(int) (gamecontext.cpp:1159)
==463034==    by 0x1DEB84: CServer::ProcessClientPacket(CNetChunk*) (server.cpp:1458)
==463034==    by 0x1E1E60: CServer::PumpNetwork() (server.cpp:2179)
==463034==    by 0x1E3708: CServer::Run() (server.cpp:2528)
==463034==    by 0x1E87F2: main (server.cpp:3525)
==463034== 
==463034== Conditional jump or move depends on uninitialised value(s)
==463034==    at 0x219909: CGameContext::SendChat(int, int, char const*, int, int) (gamecontext.cpp:408)
==463034==    by 0x21D498: CGameContext::OnClientEnter(int) (gamecontext.cpp:1159)
==463034==    by 0x1DEB84: CServer::ProcessClientPacket(CNetChunk*) (server.cpp:1458)
==463034==    by 0x1E1E60: CServer::PumpNetwork() (server.cpp:2179)
==463034==    by 0x1E3708: CServer::Run() (server.cpp:2528)
==463034==    by 0x1E87F2: main (server.cpp:3525)
==463034== 
```
Might be related to https://github.com/ddnet/ddnet/issues/2370

**Comment by def-** [comment]
Does not fix #2370.

**Comment by heinrich5991** [comment]
Reverted this with #2436 because it caused troubles with CI.
